project/MinnieKenny.scala

```diff
@@ -1,7 +1,7 @@
 import sbt.Keys._
 import sbt._
 import complete.DefaultParsers._
-import sbt.ProcessLogger
+//import sbt.ProcessLogger
```

Comment:
( completely optional ) @kshakir do you happen to know why `ProcessLogger` is no longer available in current SBT?

I can't tell how essential it is so I just commented out the affected code for now.

Reply:
Two lines to update:

1. Try these as your new imports:

```scala
import sbt.Keys._
import sbt._
import complete.DefaultParsers._
import scala.sys.process._ // <-- this is the new import
```

2. Use this new `Logger` type

```scala
def runOnce(log: Logger, args: Seq[String]): Unit = {
```

Reply:
Thank you! Saved me from a rabbit hole in a very unfamiliar area of SBT.